frontend/app/routes.tsx

```diff
@@ -36,7 +36,7 @@ export const routes = createRoutesFromElements(
       <Route path="data-entry" element={<DataEntryLayout />}>
         <Route index element={<DataEntryHomePage />} />
         <Route path=":pollingStationId" element={<PollingStationLayout />}>
-          <Route index element={<Navigate to="./recounted" replace />} />
+          <Route index element={null} />
```

Comment:
From just browsing "routes.tsx" perspective this is quite weird. how do you find the component responsible for rendering the index?

Reply:
I've added a comment in 3a6fd373c8c4648004a4989e1ea6870ebe9ae8b5, does that make it more clear?